.github/workflows/awsfulltest.yml

```diff
@@ -10,7 +10,7 @@ on:
 jobs:
   run-tower:
     name: Run AWS full tests
-    if: github.repository == 'nf-core/vcftomaf'
+    if: github.repository == 'qbic-pipelines/vcftomaf'
```

Comment:
no full tests set up

Reply:
Should I set this up, or remove the full-size test?

Reply:
remove, we don't have credits at qbic and can't add it to nf-core since it is not an nf-core piplein